### DISCUSSION TITLE
Is there any way to get targets or rushing attempts as a part of stats?

### DISCUSSION THREAD

**Discussion by noahantoun** [cwendt94/espn-api#263] [ESPN API] [answered]
Hi, Is there any way to get targets or rushing attempts as a part of stats?

**Comment by cwendt94** [comment, marked as answer]
Hey @noahantoun, I just looked at some of the unmapped stats and found the rushing attempts and targets. I can definitely add them to the stats map.

**Reply by cwendt94** [reply]
Stats should now be in package v0.18.4!

**Reply by noahantoun** [reply]
That is awesome! Thanks!
